src/main/java/sg/ncl/testbed_interface/HttpUtils.java

```diff
@@ -0,0 +1,51 @@
+package sg.ncl.testbed_interface;
+
+/**
+ * Created by dcsjnh on 11/29/2016.
+ *
+ * References:
+ * [1] https://github.com/23/resumable.js/blob/master/samples/java/src/main/java/resumable/js/upload/HttpUtils.java
+ */
+public class HttpUtils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)